pycc/cclambda.py

```diff
@@ -120,7 +150,77 @@ def solve_lambda(self, e_conv=1e-7, r_conv=1e-7, maxiter=100, max_diis=8, start_
             Gvv = self.build_Gvv(t2, l2)
             r1 = self.r_L1(o, v, l1, l2, Hov, Hvv, Hoo, Hovvo, Hovov, Hvvvo, Hovoo, Hvovv, Hooov, Gvv, Goo)
             r2 = self.r_L2(o, v, l1, l2, L, Hov, Hvv, Hoo, Hoooo, Hvvvv, Hovvo, Hovov, Hvvvo, Hovoo, Hvovv, Hooov, Gvv, Goo)
-
+   
+            if isinstance(r1, torch.Tensor):
+                del Goo, Gvv, Hoo, Hvv, Hov, Hovvo, Hovov, Hvvvo, Hovoo, Hvovv, Hooov
+         
+            if self.ccwfn.model == 'CC3':                                       
+                if isinstance(t1, torch.Tensor):
```

Comment:
What is the status of the GPU implementation for this code?

Reply:
Thanks for your comments, Dr. Crawford! It's not included here. I was thinking about having the GPU implementation and the single-precision modification in a separate PR after the equations being reviewed. But I can also add it after all the comments about the CPU code are resolved, I just wanted to make sure that the code works properly first if that makes sense to you.